src/sentry/incidents/endpoints/organization_alert_rule_index.py

```diff
@@ -33,15 +45,72 @@ def post(self, request, organization):
         """
         Create an alert rule
         """
+        # TODO: Can save all created objects in a list for a rollback in case there are any errors. Just pass list to rollback function.
+        # TODO: Use create_alert_rule, create_alert_rule_trigger, and create_alert_rule_trigger_action?
```

Comment:
Using the serializers is probably fine here, since they handle all the validation for you.